addon/modifiers/did-intersect.js

```diff
@@ -30,6 +38,7 @@ export default class DidIntersectModifier extends Modifier {
   }
 
   didUpdateArguments() {
+    this._resetIntersectionCounters();
```

Comment:
I don't think we actually want to do this. The only thing they could change is the allowed max, but we wouldn't want to reset the internal counter because of this, IMO. That could lead to some really weird unintended functionality where events fire more times than expected.

Reply:
removed